### ISSUE TITLE
Support 3DS?

### ISSUE THREAD

**Issue by rhaleblian** [rhaleblian/dslibris#33]
* see what the user experience for homebrew is generally like
* look at developer requirements
* check out what a dev environment would be like

**Comment by rhaleblian** [comment]
keywords: citra devkitpro

**Comment by rhaleblian** [comment]
#36 3DS users probably use package managers that don't with dslibris.

**Comment by Barrettloganj13** [comment]
it would be a improvement for 3ds because it runs on 3ds

**Comment by rhaleblian** [comment]
#11

**Comment by Barrettloganj13** [comment]
Yes it does work on 3ds but it could be better with more proccessing power that the dsi/3ds have

**Comment by rhaleblian** [comment]
#43 #42

**Comment by rhaleblian** [comment]
#28

**Comment by rhaleblian** [comment]
I've now finally seen it running myself.  it's worthwhile to port, and tractable to try.

**Comment by rhaleblian** [comment]
The plan is just fixes for minimal viability on 3DS.  New features are a new app written for 3DS.
